KITE-Extras/src/main/java/io/cosmosoftware/kite/imgprocessing/OCR.java

```diff
@@ -1,105 +0,0 @@
-/*
```

Comment:
can we keep this?
I'm using on load testing

Reply:
sure, I though this was removed somehow, I'll put it back in